typescript-loops/main.ts

```diff
@@ -0,0 +1,103 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen(): Array<number> {
+  const numbers = [];
+  let currentNumber = 1;
+  while (currentNumber < 11) {
+    numbers.push(currentNumber);
+    ++currentNumber;
+  }
+  return numbers;
+}
+
+let results1 = [];
+results1 = getNumbersToTen();
+console.log(results1);
```

Comment:
You line 19 and 20 are not really necessary. You have assigned an array empty array literal to a variable, and then reassigned the return value of your `getNumbersToTen` function back to that variable. 

Since `getNumbersToTen` is returning an array already, you could just log the return of calling the `getNumbersToTen` function or initialize the variable with the return of the function call rather than assigning an empty array to get overwritten.